### ISSUE TITLE
Is the polyfill working right now?

### ISSUE THREAD

**Issue by viktorix** [WICG/focus-visible#185]
Was checking out demo, .focus-visible and data-focus-visible-added are added on mouse click, 

This is on Chrome 69.0.3497.100, Firefox 62.0.3. 
No console errors.

Demo I'm referring to:
https://wicg.github.io/focus-visible/demo/

Thanks.

**Comment by robdodson** [comment]
It seems to be working fine for me? Can you check your console and let me know if you see any errors.

**Comment by viktorix** [comment]
Ah that does make sense now. I would suggest adding this info to "how it works" section in readme, for people like me :laughing: 

Thanks Rob.

**Comment by oatymart** [comment]
I agree, I wasted too much time trying to debug why it was just not happening on text inputs. The relevant info is a footnote of a linked document, whereas it could be in big letters, on your main readme and your demo page @robdodson

**Comment by robdodson** [comment]
@oatymart would you care to do a PR to add the info to the "How it works" section?

**Comment by oatymart** [comment]
I will give it a shot when I (fingers crossed) find the time this week or next!
